src/compass/s1_cslc_qa.py

```diff
@@ -63,29 +65,41 @@ def compute_CSLC_raster_stats(self, cslc_h5py_root, bursts):
                 # create dict to store real/imaginary stat items
                 pol_dict[real_imag] = {}
 
+                # create HDF5 group for real/imaginary stats of current
+                # polarization
+                stats_path = f'{GRID_PATH}/stats/{pol}/{real_imag}'
+                stats_group = cslc_h5py_root.require_group(stats_path)
+
                 # add description for stat items
                 qa_item_desc = f'{real_imag} part of geoocoded SLC'
 
                 vals = [cstat_member.mean, cstat_member.min,
                         cstat_member.max, cstat_member.sample_stddev]
-                # write stats to HDF5
+                # save stats to dict and write to HDF5
                 for val_name, val in zip(self.stat_names, vals):
+                    desc = f'{val_name} of {qa_item_desc}'
                     pol_dict[real_imag][val_name] = value_description_dict(
-                        val, f'{val_name} of {qa_item_desc}')
+                        val, desc)
+                    add_dataset_and_attrs(stats_group, Meta(val_name, val,
+                                                            desc))
 
 
     def compute_correction_stats(self, cslc_h5py_root):
         '''
-        Compute correction stats
+        Compute correction stats. Stats written to HDF5 and saved to class dict
+        for later JSON output
 
         Parameters
         ----------
         cslc_h5py_root: h5py.File
             Root of CSLC HDF5
         '''
+        corrections_path = f'{ROOT_PATH}/corrections'
+
         # compute and save corrections stats
         corrections = ['bistatic_delay', 'geometry_steering_doppler',
-                       'azimuth_fm_rate_mismatch']
+                       'azimuth_fm_rate_mismatch', 'los_ionospheric_delay',
+                       'los_solid_earth_tides']
```

Comment:
Maybe I've missed something, but are we going to include troposphere correction using weather model?

Reply:
d193f14 addresses this